lib/ulid.ex

```diff
@@ -1,9 +1,9 @@
 defmodule Ulid do
-  def generate(timestamp \\ System.system_time(:milli_seconds)) do
+  def generate(timestamp \\ System.system_time(:millisecond)) do
```

Comment:
interesting, whats the advantage of using `millisecond` ?

Reply:
this change just removes annoying warning which says that `:milli_seconds` is deprecated